Packs/CommonScripts/Scripts/UnzipFile/UnzipFile.yml

```
@@ -11,6 +11,12 @@ args:
   name: password
   required: false
   secret: true
+- default: false
+  description: Password to protect the ZIP file, inserted as a non sensative argument.
+  isArray: false
+  name: non_sensetive_password
```

the name differs from the code
```suggestion
  name: nonsensetive_password
```